app/javascript/utils/index.ts

```diff
@@ -73,7 +73,8 @@ export const clickTracking = (affiliate: string, module: string, query: string,
       module_code: module,
       position,
       query,
-      vertical
+      vertical,
+      type: 'click'
```

Comment:
While setting up AT I discovered another undocumented requirement based on current functionality.  All current SERP click tracking has a "type" of 'click' (as opposed to 'search' for search logging).